lib/cisco_node_utils/ace.rb

```diff
@@ -50,126 +41,106 @@ def ace_get
       regexp.match(str)
     end
 
-    # Create a hash of all aces under give acl_name.
+    # Create a hash of all aces under a given acl_name.
     def self.aces
-      afis = %w(ip ipv6)
+      afis = %w(ipv4 ipv6)
       hash = {}
       afis.each do |afi|
-        acls = config_get('acl', 'all_acls', afi: afi)
+        afi_cli = Acl.afi_cli(afi)
+        acls = config_get('acl', 'all_acls', afi: afi_cli)
```

Comment:
I didn't because I need this value in more than one place. But what the heck sounds good.